model.py

```diff
@@ -271,7 +274,12 @@ def __init__(self, ts, name=None):
             self.ts.mutations_node, minlength=self.ts.num_nodes
         )
 
+    @property
+    def file_uuid(self):
+        return self.ts.file_uuid
+
     @cached_property
```

Comment:
right, so there's two layers of caching here. I had to think a bit about what this means - so the first time the property is called, we fall through to the disk_cache version, and then afterwards (within that process) it'll be cached by functools.